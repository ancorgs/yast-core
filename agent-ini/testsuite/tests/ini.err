[yast2-agent-ini] IniParser.cc(parse_helper):XXX 30: Extra characters: <aaa>
[yast2-agent-ini] IniParser.cc(parse_helper):XXX 31: Extra characters: <bbb>
[yast2-agent-ini] IniParser.cc(parse_helper):XXX 32: Extra characters: <ccc>
[yast2-agent-ini] IniParser.cc(parse_helper):XXX 39: Extra characters: </ccc>
[yast2-agent-ini] IniParser.cc(parse_helper):XXX 40: Extra characters: </bbb>
[yast2-agent-ini] IniParser.cc(parse_helper):XXX 41: Extra characters: </aaa>
