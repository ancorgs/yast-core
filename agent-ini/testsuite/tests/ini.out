(["ODBC Data Sources", "OpenLink", "Default", "Huhu", "Haha"])
;
;  odbc.ini
;
; comment on line with section
[ODBC Data Sources]
OpenLink=OpenLink (MT)

[OpenLink]
Driver=/home/openlink/lib/oplodbc.so.1
Description=Sample OpenLink MT DSN
; comment on some other line
Host=localhost
ServerType=Oracle 8
FetchBufferSize=99
UserName=
Password=
Database=
ServerOptions=
ConnectOptions=
Options=
ReadOnly=no

[Default]
Driver=/home/openlink/lib/oplodbc.so.1

{comment}
Value=AAABBB

;; huhu hoho hehe
;; huuuu huuuu
;; Buuu buuu
Name=Name aaa
[Huhu]
[Haha]
aa=bb

habakuk=aaa
